README.md

```diff
@@ -8,6 +8,8 @@
 - Add a book.
 - Remove a selected book.
 
+![screenshot](./Captura.PNG)
+
```

Comment:
Your README looks great, but your MIT license link is not valid.

- [ ] please fix it;